source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.csproj

```diff
@@ -26,10 +26,11 @@
     <PlatformTarget>x64</PlatformTarget>
     <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
-    <OutputPath>..\..\..\..\..\build\$(Configuration)\$(Platform)</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
+    <OutputPath>..\..\..\..\build\Release\x64\</OutputPath>
+    <DefineConstants>TRACE;CODE_ANALYSIS</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
same comment here. In addition, should the configuration be hard coded here?

Reply:
Done. Separate commit.

Reply:
Which configuration?

Reply:
$(Configuration)$(Platform) - this is a mistake I think

Reply:
It evaluates to the same. In many of the projects $(Configuration)$(Platform) is used, what it evaluates to in this section is controlled by 
```
<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|x64' ">
```
It doesn't really matter, but it would be nice to be consistent (could be that different versions of Visual Studio was used to create, and that the other version don't bother to change.

Reply:
I've fixed it now in all proj files.